.github/workflows/ort/action.yml

```diff
@@ -0,0 +1,103 @@
+name: "Run OSS Review Toolkit"
+description: "Runs OSS Review Toolkit & generates SBoMs"
+inputs:
+  build-artifacts:
+    description: |
+      Build Artifact paths to include into SBoM.
+      May contain a glob pattern or list of paths separated by a newline.
+    required: false
+    default: ""
+  report-formats:
+    description: "ORT Report Formats"
+    required: true
+  fail-on-violation:
+    description: "Whether to fail on violation."
+    required: false
+    default: false
+  upload-reports:
+    description: "Whether to upload all reports"
+    required: false
+    default: false
+  version:
+    desctiption: "Elixir Version (Tag / SHA)"
+    required: true
+
+outputs:
+  results-path:
+    description: "See oss-review-toolkit/ort-ci-github-action action"
+    value: "${{ steps.ort.outputs.results-path }}"
+  results-notice-path:
+    description: "Path to resulting NOTICE file"
+    value: "${{ steps.ort.outputs.results-path }}/NOTICE_DEFAULT"
```

Comment:
Just checking if we need this or is this an artifact of the previous NOTICE generation we now want removed?

Reply:
Good catch, it's no longer needed.